lib/widgets/atoms/inbox_list_tile.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+import 'package:mm_flutter_app/constants/app_constants.dart';
+import 'package:mm_flutter_app/utilities/utility.dart';
+import 'package:mm_flutter_app/widgets/atoms/notification_bubble.dart';
+
+class InboxListTile extends StatelessWidget {
+  final String? avatarUrl;
+  final String fullName;
+  final DateTime date;
+  final String? message;
+  final int notifications;
+  final bool highlightMessage;
+  final bool simplifyDate;
+  final String? datePrefix;
+
+  const InboxListTile({
+    super.key,
+    this.avatarUrl,
+    required this.fullName,
+    required this.date,
+    this.message,
+    this.notifications = 0,
+    this.highlightMessage = false,
+    this.simplifyDate = false,
+    this.datePrefix,
+  });
+
+  String _getDate(BuildContext context) {
+    if (!simplifyDate) {
+      return AppUtility.timestampDateFormat(context, date);
+    } else if (datePrefix == null) {
+      return AppUtility.simplePastDateFormat(
+        context,
+        date,
+        capitalize: true,
+      );
+    } else {
+      return '$datePrefix ${AppUtility.simplePastDateFormat(
+        context,
+        date,
+        capitalize: false,
+      )}';
+    }
+  }
```

Comment:
This is really cool functionality. I wonder if this can be an app-level function in the future, wherein wherever we have "date" elements we can change the format based of user preferences instead of something tied to inbox list tile specifically. Not blocking.